llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -13068,6 +13083,17 @@ SDValue DAGCombiner::foldPartialReduceMLAMulOp(SDNode *N) {
   SDValue LHSExtOp = LHS->getOperand(0);
   EVT LHSExtOpVT = LHSExtOp.getValueType();
 
+  // Return 'select(P, Op, splat(0))' if P is nonzero,
+  // or 'P' otherwise.
+  auto tryPredicate = [&](SDValue P, SDValue Op) {
```

Comment:
very nit: lambdas should be named like other locals: 
```suggestion
  auto TryPredicate = [&](SDValue P, SDValue Op) {
```